tests/functional/spec/features/submit_page/submit_page_summary.spec.js

```diff
@@ -36,6 +36,15 @@ describe("Submit Page with Summary", () => {
     expect($(RadioPage.bacon()).isSelected()).to.be.true;
   });
 
+  it("Given a questionnaire with a summary has been completed and a summary page edit link is clicked, when I click previous, then it should return to the summary", () => {
+    completeAllQuestions();
+
+    $(SubmitPage.radioAnswerEdit()).click();
+    $(RadioPage.previous()).click();
+
+    expect(browser.getUrl()).to.contain(SubmitPage.pageName);
+  });
```

Comment:
We should also test that this only happens on the first page you land on. I don't think we have a test to assert the next page's previous link doesn't go to summary, but just the previous question, essentially asserting the return_to isn't persisted. You need to change path to test this. Not sure if this change allow for path changes, if not the `test_section_summary` one does.